Tools/TestWebKitAPI/Tests/WebKitCocoa/WKAttachmentTests.mm

```diff
@@ -1628,6 +1643,34 @@ @implementation FileWrapper
     EXPECT_TRUE([originalData isEqualToData:pastedAttachmentInfo.data]);
 }
 
+static bool triedToLoadThumbnail;
+static void _generateBestRepresentationForRequest(id, SEL)
+{
+    triedToLoadThumbnail = true;
+}
+
+TEST(WKAttachmentTests, CutAndPasteAttachmentBetweenWebViews)
+{
+    InstanceMethodSwizzler quickLookSwizzler {
+        getQLThumbnailGeneratorClass(),
+        @selector(generateBestRepresentationForRequest:completionHandler:),
+        reinterpret_cast<IMP>(_generateBestRepresentationForRequest)
+    };
```

Comment:
It might be a good idea to put this swizzler below the
```
[webView _synchronouslyExecuteEditCommand:@"Cut" argument:nil];
```
..line below. If we eventually fix `-_insertAttachmentWithFileWrapper:` to automatically request QL thumbnails for the inserted attachment, then I think this test might become a false positive (since we'll get two calls to the swizzled `_generateBestRepresentationForRequest`).

Reply:
Ok - I reset the bool variable to false below to avoid that but probably safe to move the swizzler as well.

Thanks for the review!